include/SDL3/SDL_stdinc.h

```diff
@@ -1073,6 +1257,23 @@ extern SDL_DECLSPEC void * SDLCALL SDL_memcpy(SDL_OUT_BYTECAP(len) void *dst, SD
     { SDL_COMPILE_TIME_ASSERT(SDL_copyp, sizeof (*(dst)) == sizeof (*(src))); }             \
     SDL_memcpy((dst), (src), sizeof(*(src)))
 
+/**
+ * Copy memory.
+ *
+ * It is okay for the memory regions to overlap.
+ * If you are confident that the regions never overlap, using SDL_memset() may improve performance.
```

Comment:
```suggestion
 * If you are confident that the regions never overlap, using SDL_memcpy() may improve performance.
```

I'm sure randomly zeroing memory will improve performance sometimes, so technically what you said is true, but I don't think it's what you meant :-)